packages/backup-restore-operator/charts/templates/s3secret.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.s3 -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-s3
```

Should this be changed to the following?

```suggestion
  name: {{ .Chart.Name }}-s3
```

---

why can't we use the release name?

---

`.Chart.Name` is globally fixed, whereas the `.Release.Name` is determined by the user input.

Unless there's a use case for a user deploying multiple backup operators onto the same cluster with different Release names (which might require some more changes due to CRDs being in the `templates/` directory instead of the `crd/` directory or a separate CRD chart), I think it would be preferable to use the `.Chart.Name`